src/targets/cpu/lowering.cpp

```diff
@@ -593,13 +593,33 @@ struct cpu_unary
 {
     Op op;
     std::string name() const { return op.name(); }
-    shape compute_shape(const std::vector<shape>& inputs) const { return inputs.front(); }
+    shape compute_shape(const std::vector<shape>& inputs) const
+    {
+        if(inputs.at(0).packed())
```

Comment:
It might be better to assign a variable to inputs.at(0), like in unary.cpp

Reply:
fixed